assets/src/apps/authoring/components/Flowchart/sidebar/FlowchartSidebar.tsx

```diff
@@ -76,7 +77,9 @@ const SelectedScreen: React.FC<{ screen: IActivity }> = ({ screen }) => {
   };
 
   const paths = screen.authoring?.flowchart?.paths || [];
-  const addPathDisabled = questionType === 'none' && paths.length > 0;
+  const screenType = screen.authoring?.flowchart?.screenType || 'none';
+  const addPathDisabled =
+    screenType === QuestionTypeMapping.HUB_SPOKE || (questionType === 'none' && paths.length > 0);
```

Comment:
User should not update the flow chart logic from the flow chart panel, so we disable it.